docs/03-Cloud-Native-Journey/02-learning-goals.adoc

```diff
@@ -2,90 +2,118 @@
+==== LG 3-1: Understand the Cloud Native Concept
 
-* Interfaces for administration and configuration
+Software architects understand the benefits and goals of the Cloud Native concept. They are familiar with the advantages of the concept, particularly through:
+
+* Unified interfaces for administration and configuration
 * Resource abstraction
 * Delivery artifacts
-* Isolation of the application, with respects to the host, etc.
+
 
 [[LG-3-2]]
-==== LG 3-2: Container Manager
-* Understand the abstraction concepts of current container managers and know patterns and best practices for how to apply non-functional requirements (NFR) fulfillment to them. For example:
-** Finite workloads such as (cron) jobs
-** Health Check and Self Healing
-** Scaling and Load Balancing
-** Placement
-* Know the principles of an overlay network (VLAN) for containers and container managers and understand how to isolate different services or systems. E.g., implement a distributed firewall to isolate:
-** different clients
-** test systems in the context of a CI/CD
+==== LG 3-2: Understand basic container concepts
 
-[[LG-3-3]]
-==== LG 3-3: Container
-Understand the Linux kernel isolation mechanisms that apply to containers (container security), including:
+Software architects understand the technical basics and the added value of containers and know the difference between conventional deployment and deployment with containers.
+
+They understand how the use of containers leads to a better decoupling of responsibilities and thus to a more effective organization.
+
+Software architects understand Linux kernel isolation mechanisms that apply to containers (container security), including:
 
+* Namespaces and cgroups
 * Capabilities
-* Privileged Containers
-* AppArmor
+
+They also understand the implications, usage scenarios, and threats of privileged containers.
+
+[[LG-3-3]]
+==== LG 3-3: Understand Container Manager Abstraction Concepts
+
+Software architects understand the abstraction concepts of current container managers and know patterns and best practices on how to apply non-functional requirements (NFR) fulfillment to them. For example:
+
+* Finite workloads such as (cron) jobs.
+* Health Check and Self Healing
+* Scaling and load balancing
+* Placement
+
+Software architects know the principles of an overlay network (VLAN) for containers and container managers and understand how different services or systems are isolated. They are familiar with requirement scenarios such as implementing a distributed firewall for isolating:
+
+
+* Multiple tenants
+* Test systems in the context of CI/CD
 
 [[LG-3-4]]
-==== LG 3-4: Container Linux
-understand how current core/container Linux distributions are optimized for running containers, what advantages they offer (including kernel and distribution updates), and what the consequences are for the applications they run.
+==== LG 3-4: Understand the benefits of specialized Linux distributions for operating containers
```

Comment:
Manchmal haltet ihr euch daran, jedes Wort in der Überschrift groß zu schreiben, manchmal schreibt ihr es wie einen Satz. Hier wäre es schön, eine einheitliche Form zu haben. Siehe auch https://capitalizemytitle.com/style/APA/

Reply:
✔️